### DISCUSSION TITLE
Multiple locations for locales

### DISCUSSION THREAD

**Discussion by rune-hansen** [i18next/next-i18next#2167] [Q&A]
Is it possible to specify multiple locations for the locales. Something like having "localePath" accept an array?

We have our global locales in a core package that is used by all apps and only the app specific locales are under the src/i18n/locales folder

    .
    ├── FinanceManager - (App)/
    │   ├── ...
    │   └── src/
    │       └── i18n/
    │           └── locales/
    │               ├── en/
    │               │   ├── common.json
    │               │   ├── invoices.json
    │               │   ├── journals.json
    │               │   └── approvals.json
    │               └── it/
    │                   ├── common.json
    │                   ├── invoices.json
    │                   ├── journals.json
    │                   └── approvals.json
    │
    └── core-frontend-functionality - (Package)/
        ├── ...
        └── src/
            └── i18n/
                └── locales/
                    ├── en/
                    │   ├── common.json - (Empty)
                    │   └── core.json
                    └── it/
                        ├── common.json - (Empty)
                        └── core.json

**Comment by knoefel** [comment]
I'm facing the same situation right now, where the translations are located at different paths inside the application. I have 2 core libs (shared translations and shared ui components) from which i want to load the translations and also feature libs with its own translation. 

@adrai Is there a way to achieve this with the current api?

I tried setting the `localePath` for the feature lib and also the `resources` prop for the shared core translations, but it seems `next-i18next` only respects one of them not both props at the same time.

@rune-hansen How did you solve it?

Thanks in advance for any tipp!

**Reply by adrai** [reply]
no

**Reply by knoefel** [reply]
@adrai Thanks for reaching out! Are there any workarounds i could try? I've read in another discussion that i can modify the `initialI18nStore` and pass there the translations loaded via `resources`. But i guess this would increase the bundle size for each page, because i would need to add translations for every locale there? Thanks again for the quick reply!
